micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java

```diff
@@ -103,7 +118,7 @@ public void initialize() {
                 }
                 long executionLatency = hystrixCommandCompletion.getExecutionLatency();
                 if (executionLatency >= 0) {
-                    Timer.builder("hystrix.latency.total")
+                    Timer.builder(NAME_HYSTRIX_LATENCY_EXECUTION)
```

Comment:
How the heck are you finding all these? Much appreciated!

Reply:
I want to know the same thing.

Reply:
@checketts @jkschneider Just by watching this project 😅

Reply:
If you haven't been following the work done by @izeye in spring-boot, do, you'll be even more amazed.